SmalltalkSources/SBE-Testing.package/SBEtest.class/instance/sourceLines..st

```diff
@@ -16,5 +16,6 @@ sourceLines: lines
 				self unescapeLatex: (self systemVersionString >= (matcher subexpression: 2)
 					ifTrue: [matcher subexpression: 3]
 					ifFalse: [matcher subexpression: 5])]].
+	sourceLines := sourceLines collect: [:line | line withBlanksTrimmed].
```

Comment:
Did you insert this extra line instead of removing the trailing space in the tex file? ;-)

Reply:
Of course, fail-safety is helpful 👍 But how do you think about adding an .editorconfig to this repo to avoid this in the future?

Reply:
.editorconfig sounds like a good idea!

Reply:
See #114. :-)